app/assets/stylesheets/application.css

```diff
@@ -0,0 +1,13 @@
+/* The theme's stylesheet initializes tailwindcss and imports the theme's default styles.
+
+   The $ThemeStylesheet path is evaluated via a postcss resolver, which finds the theme's
+   main stylesheet within the theme's gem.
+   
+   To control the order of the tailwindcss imports further, you can eject the theme's
+   stylesheet locally using the following command, which will take precedence.
+   > rake bullet_train:themes:light:eject_css
+
+   Read more: https://bullettrain.co/docs/stylesheets
+*/
+@import "$ThemeStylesheet";
+@import "application.custom";
```

Comment:
Why do we need the separate `application.custom` file?

I'm thinking of the pattern we've established in the rest of the starter repo like this:

```ruby
class User < ApplicationRecord
  include Users::Base
end
```

My meaning being that in the starter repo, users own the resulting files but we integrate by pulling stuff in from the rest of Bullet Train. Having a separate `application.custom.css` breaks that (albeit subtle) convention. Versus `application.css` having a theme import while also containing custom extensions makes more sense to me. What do you think?

Reply:
Yeah, I was having second thoughts about `application.custom` for the same reasons as we're breaking the pattern. (And I thought you'd be bringing it up @kaspth).

Even [`app/javascript/application.js`](https://github.com/bullet-train-co/bullet_train/blob/main/app/javascript/application.js) is where we encourage people to edit after our own imports.

https://github.com/bullet-train-co/bullet_train/blob/ebc520f1bb56d990aa263b88be69ff08429283d4/app/javascript/application.js#L22-L34

Will remove the second file.

Reply:
@pascallaliberte oh cool! I didn't even think about `app/javascript/application.js`, but looks like there's a pattern we can follow in there with similar wording.

> (And I thought you'd be bringing it up @kaspth).

You know me too well 😄